libyul/optimiser/SSAValueTracker.cpp

```diff
@@ -37,7 +37,7 @@ void SSAValueTracker::operator()(VariableDeclaration const& _varDecl)
 {
 	if (_varDecl.variables.size() == 1)
 		setValue(_varDecl.variables.front().name, _varDecl.value.get());
-	else
+	else if (!_varDecl.value)
```

Comment:
What happens to variable declarations with more than one variable that do have a value?

Reply:
They are ignored (i.e. treated the same way as variables that are re-assigned at some point) because there is no syntactic way to refer to the value component.